src/utils/jsHelper/helper.tsx

```
@@ -412,3 +413,18 @@ export function checkEligibilityCriteria({
       return false;
   }
 }
+export function getIncomeRangeValue(annualIncome: string): string {
+  const income = parseFloat(annualIncome);
+  if (isNaN(income)) return undefined;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)